hazelcast-client/src/main/java/com/hazelcast/client/impl/statistics/Statistics.java

```diff
@@ -61,14 +62,10 @@
     public static final HazelcastProperty PERIOD_SECONDS = new HazelcastProperty("hazelcast.client.statistics.period.seconds", 3,
             SECONDS);
 
-    private static final String NEAR_CACHE_CATEGORY_PREFIX = "nc.";
-    private static final String FEATURE_SUPPORTED_SINCE_VERSION_STRING = "3.9";
+    private static final String FEATURE_SUPPORTED_SINCE_VERSION_STRING = "3.11"; //TODO later set to 3.12
     private static final int FEATURE_SUPPORTED_SINCE_VERSION = BuildInfo.calculateVersion(FEATURE_SUPPORTED_SINCE_VERSION_STRING);
-    private static final char STAT_SEPARATOR = ',';
-    private static final char KEY_VALUE_SEPARATOR = '=';
-    private static final char ESCAPE_CHAR = '\\';
 
-    private final MetricsRegistry metricsRegistry;
+    private final ProbeRegistry.ProbeRenderContext probeRenderContext;
```

Comment:
Same here. Please don't use inner interfaces. We don't do this in the rest of the system.